redbot/cogs/downloader/downloader.py

```diff
@@ -53,6 +53,8 @@ def __init__(self, bot: Red):
         self._create_lib_folder()
 
         self._repo_manager = RepoManager()
+        self._ready = asyncio.Event()
+        self._init_task = bot.loop.create_task(self.initialize())
```

Comment:
This line probably shouldn't be part of a constructor method as it is possible for the bot to fail to add the cog.